.ebextensions/01_rails_db_setup.config

```diff
@@ -0,0 +1,10 @@
+files:
+  "/opt/elasticbeanstalk/hooks/appdeploy/post/01_rails_db_setup.sh":
+    mode: "000755"
+    owner: root
+    group: root
+    content: >
+      #!/usr/bin/env bash
+      cd /var/app/current
+      DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bin/rails db:setup'
```

Comment:
In production mode, Elastic Beanstalk won't let you run commands that muck with the database. The error message helpfully tells you how to get around that by using `DISABLE_DATABASE_ENVIRONMENT_CHECK`.

Reply:
how does EB know that you are mucking with the database?

Reply:
Perhaps I misspoke. Looks to be Rails protecting the database in `production` mode:

```sh
ActiveRecord::ProtectedEnvironmentError: You are attempting to run a destructive action against your 'production' database.
If you are sure you want to continue, run the same command with the environment variable:
DISABLE_DATABASE_ENVIRONMENT_CHECK=1
bin/rails:4:in `require'
bin/rails:4:in `<main>'
Tasks: TOP => db:schema:load => db:check_protected_environments
(See full trace by running task with --trace)
```